multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/configuration/FlowableConfiguration.java

```diff
@@ -73,22 +75,32 @@ public SpringProcessEngineConfiguration processEngineConfiguration(DataSource da
 
     @Inject
     @Bean
-    public AsyncExecutor jobExecutor(ApplicationConfiguration configuration, String jobExecutorId) {
-        DefaultAsyncJobExecutor jobExecutor = new DefaultAsyncJobExecutor();
-        scale(configuration, jobExecutor);
+    public AsyncExecutor jobExecutor(DefaultAsyncTaskExecutor asyncTaskExecutor, String jobExecutorId) {
+        DefaultAsyncJobExecutor jobExecutor = new DefaultAsyncJobExecutor() {
+            @Override
+            protected void initAsyncJobExecutionThreadPool() {
+                asyncTaskExecutor.start();
+                this.taskExecutor = asyncTaskExecutor;
+                this.shutdownTaskExecutor = true;
+            }
+        };
```

Comment:
I have implemented that in this way, because when using `jobExecutor.setTaskExecutor`, this init method (check its implementation) does not start the task executor if it is already set.

Reply:
As I can see, `AsyncExecutor` interface has `start()` method, did you try in this way?